eng/build.ps1

```diff
@@ -7,6 +7,7 @@ Param(
   [string][Alias('c')]$configuration = "Debug",
   [string][Alias('f')]$framework,
   [string]$vs,
+  [string]$vspath,
```

Comment:
Why do we need this parameter? By default we use the VS which is associated with the `.sln` extension. Isn't that sufficient? If you wanna run on different VS versions you can change the association on your OS.

Reply:
The default for .sln is to use the VS Version Selector stub which then opens the "right" VS - I thought it might be a good idea to not force people to hard selected version. But maybe it's a moot point... I'm happy to remove this if others think it's not necessary

Reply:
I would prefer to not add this parameter as part of this PR and discuss in a separate issue if we want such an extension point.